### PR TITLE
O3-1628 Get list of all queues

### PR DESCRIPTION
**Pull request by CynthiaKamau**
O3-1628 Get list of all queues - Added ability to retrieve all queues

### CODE REVIEW

**Comment by dkayiwa** [comment]
@CynthiaKamau do you mind creating a ticket for these changes in this JIRA project? https://issues.openmrs.org/projects/PQ

**Comment by CynthiaKamau** [comment]
> @CynthiaKamau do you mind creating a ticket for these changes in this JIRA project? https://issues.openmrs.org/projects/PQ

Done

**Comment by dkayiwa** [comment]
Can you include the ticket url as advised here? https://wiki.openmrs.org/display/docs/Pull+Request+Tips

**Review by corneliouzbett — COMMENTED** [review]
Thanks, @CynthiaKamau for working on this.
Could you please add tests for this change otherwise LGTM.

**Comment by CynthiaKamau** [comment]
> Thanks, @CynthiaKamau for working on this. Could you please add tests for this change otherwise LGTM.

Done

File: omod/src/test/java/org/openmrs/module/queue/web/resources/QueueResourceTest.java [hunk 1]
```diff
@@ -152,4 +152,9 @@ public void shouldFindQueuesByLocation() {
 		assertThat(result.getTotalCount(), is(1L));
 		result.getPageOfResults().forEach(q -> assertThat(q.getLocation().getUuid(), is(LOCATION_UUID)));
 	}
+
+	@Test
+	public void shouldGetAllQueues() {
+		assertThat(queueService, notNullValue());
```
Comment (corneliouzbett) [inline omod/src/test/java/org/openmrs/module/queue/web/resources/QueueResourceTest.java:158]: Do this instead;
```java
Queue queueMock = mock(Queue.class);

when(queueService.getAllQueues()).thenReturn(Collections.singletonList(queueMock));

Collection<Queue> result = queueService.getAllQueues();
assertThat(result, hasSize(1));
assertThat(result, hasItem(queueMock));
```

**Review by corneliouzbett — CHANGES_REQUESTED** [review]

**Review by corneliouzbett — APPROVED** [review]
Thanks @CynthiaKamau!

**Pull request merged**